### PR TITLE
isUpperCase package

### PR DESCRIPTION
**Pull request by humanolaranja**
#11

### CODE REVIEW

**Comment by vorillaz** [comment]
@humanolaranja Thanks for your contribution. 

You should also upgrade the `package/plexis/package.json` 
Take a look at this PR for reference #18 .
Also, you should run `yarn test -u` in order to update the Jest snapshots.

**Comment by humanolaranja** [comment]
> @humanolaranja Thanks for your contribution.
> 
> You should also upgrade the `package/plexis/package.json`
> Take a look at this PR for reference #18 .
> Also, you should run `yarn test -u` in order to update the Jest snapshots.

Thanks, now everything is ok =D

**Comment by vorillaz** [comment]
@humanolaranja Thanks so much for contributing.
Can you please rebase your branch and resolve your conflicts?

**Comment by humanolaranja** [comment]
Yes, I'll do it now

**Comment by vorillaz** [comment]
@humanolaranja Also, can you also add your method to `docs/pages/api.md` ?
Thanks so much.

**Comment by humanolaranja** [comment]
Everything is updated now =D

File: packages/isUpperCase️/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+isUpperCase️('b') // returns false
+isUpperCase️('B') // returns true
+isUpperCase️('BCD') // returns true
+```
+
+**Aliases**
+
+```javascript
+import {isUpperCase} from 'plexis';
```
Comment (vorillaz) [inline packages/isUpperCase️/README.md]: Can you please merge these two into one line?

Comment (humanolaranja) [reply packages/isUpperCase️/README.md]: Sorry, I didn't understood

**Comment by vorillaz** [comment]
@humanolaranja Also, can you please rebase your branch ?

**Comment by humanolaranja** [comment]
> @humanolaranja Also, can you please rebase your branch ?

Done

**Pull request merged**